### PR TITLE
Consolidate Quickstart Guide links on server console

### LINKED ISSUE #5238: Consolidate "Quickstart Guide" links on Minio server console
An effort to simplify and provide clear details on the console output of Minio server (a la TLDR). 
## Current Behavior
There are 5 separate lines for each link pointing to Go, Java, Python, JavaScript and .NET separately. 

## Possible Solution
Consolidate all links to a landing document link and still keep the supported languages APIs 

## Steps to Reproduce (for bugs)

1. start minio server

## Context
Current View:
```
Object API (Amazon S3 compatible):
   Go:         https://docs.minio.io/docs/golang-client-quickstart-guide
   Java:       https://docs.minio.io/docs/java-client-quickstart-guide
   Python:     https://docs.minio.io/docs/python-client-quickstart-guide
   JavaScript: https://docs.minio.io/docs/javascript-client-quickstart-guide
   .NET:       https://docs.minio.io/docs/dotnet-client-quickstart-guide
```
Possible View:
```
Object API (Amazon S3 compatible):
   Go, Java, Python, JavaScript & .NET: https://docs.minio.io/docs/quickstart-guides
```

## Your Environment

* Version used (`minio version`):
* Environment name and version (e.g. nginx 1.9.1):
* Server type and version:
* Operating System and version (`uname -a`):
* Link to your project:

### PR DESCRIPTION
**Pull request by shruthied**
Added README file under sdks folder describing Minio SDKs along with
quickstart guides and API reference links.
Modified the code in the server startup message file accordingly.
Fixes #5238

### CODE REVIEW

**Comment by shruthied** [comment]
@nitisht  and @kannappanr plz review my changes

File: docs/sdks/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (harshavardhana) [inline docs/sdks/README.md]: Remove frivolous space above.

File: docs/sdks/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Minio SDKs
+Minio SDKs are available in different languages and contain the library, examples and documentation. Object APIs exposed by the SDKs help users build cloud-native applications for multi-cloud environments.
+
+Click on the links below to know more about the SDKs and APIs.
+
```
Comment (harshavardhana) [inline docs/sdks/README.md:5]: Frivolous space.

File: docs/sdks/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+| SDK | Quick start guide | API reference |
+|:--- |:--- |:--- |
+| `Golang` | [Quick start guide](https://docs.minio.io/docs/golang-client-quickstart-guide)| [API Reference](https://docs.minio.io/docs/golang-client-api-reference) |
```
Comment (harshavardhana) [inline docs/sdks/README.md]: s/Golang/Go

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by harshavardhana** [comment]
This PR is blocked for review until @abperiasamy takes a look, there is a bit revamping that needs to be done before we bring this in. 

@shruthied will update you with new changes once we have finalized.

**Comment by kannappanr** [comment]
After discussion, it has been decided that we will not be merging this PR, as quite a few users feel that the existing links to the SDKs is good and consolidating them is not required.

**Pull request closed without merging**